### ISSUE TITLE
genvalidity-0.11.0.2: Variable not in scope: isLineSeparator

### ISSUE THREAD

**Issue by juhp** [NorfairKing/validity#88]
```
Building library for genvalidity-0.11.0.2..                                                                                         
[2 of 4] Compiling Data.GenValidity                               
                                                                                                                                    
/var/stackage/work/unpack-dir/unpacked/genvalidity-0.11.0.2-3d6a0489512630870494f29bc8efefa8f2cf6dfa55f45eb6373e17849e5d5598/src/Data/GenValidity.hs:652:50: error:                                   
    • Variable not in scope: isLineSeparator :: Char -> Bool                                                                        
    • Perhaps you meant ‘genLineSeparator’ (line 649)                                                                               
    |                                                                                                                               
652 | genNonLineSeparator = genValid `suchThat` (not . isLineSeparator)                                                             
    |                                                  ^^^^^^^^^^^^^^^                                                              
```

**Comment by NorfairKing** [comment]
should be fixed with `validity-0.11.0.1` which I uploaded just now

**Comment by juhp** [comment]
Thanks
